### ISSUE TITLE
CC BY-NC-SA, Personal Privacy, and You

### ISSUE THREAD

**Issue by tweedge** [r-cybersecurity/list-of-security-resources-for-ukraine#22]
TL;DR: This repository contains contact information and links to contact information - which, especially where individuals are concerned, can make it hard to understand what is and isn't OK to republish. As anyone contributing to this repository (or other r/cybersecurity works) knows, this is a CC BY-NC-SA 4.0 licensed work, which allows anyone to adapt, remix, and redistribute its contents. A contributor raised that they felt I had violated a boundary of theirs, as they consider the email address to be private information (this is my error). In now-deleted issue #21, I removed their company and purged the repository history - as my initial assessment was that their company's inclusion in this repository may be incompatible with their privacy wishes.

It has since become clear that it is possible for the company's data can be included in this CC BY-NC-SA 4.0 repository, but to limit person's/company's exposure to unwanted (and unauthorized!) proliferation of their contact information, I ask that anyone who is planning to remix or adapt this work keep the following in mind:

**You can only remix and adapt what is specifically included in this repository. Where this repository links to a *separate site* - for example, a site which has a company or individual's contact information - you must abide by that site's copyright, licensing, etc. when determining whether or not *that external site's information* can be included in your remix or adaptation of this work.**

An example will help illustrate this. Consider "Company A" has listed under their contact information "Email us using the address found on `https://company-a.com/contact/`" and you see in the footer of Company A's website "Copyright (c) 2022 Company A." Your rights under CC BY-NC-SA 4.0 are:

* **OK:** Republishing content in this repository verbatim (so long as you follow the BY-NC-SA clauses).
* **OK:** Rewording the content to read "I encourage impacted companies to contact Company A at `https://company-a.com/contact/`" (so long as you follow the BY-NC-SA clauses).
* **NOT OK:** Copying the email off `https://company-a.com/contact/` and including it in this repository or elsewhere (because the external site is copyrighted - you can publish a link to that site, but you **cannot** republish its contents).

I hope this helps explain your rights while preserving the rights of the people and companies volunteering here.

Thank you for your attention & understanding in this matter.
